package.json

```diff
@@ -17,17 +17,17 @@
     "contributors:add": "all-contributors add",
     "contributors:generate": "all-contributors generate",
     "contributors:check": "all-contributors check",
-    "postinstall": "run-p install:*",
     "docs": "./scripts/docsite.sh",
+    "postinstall": "run-p -l --max-parallel 3 'install:*'",
     "install:wasi": "bash scripts/setup_wasi.sh",
-    "install:vscode-wing": "npm --prefix apps/vscode-wing install",
-    "install:tree-sitter-wing": "npm --prefix libs/tree-sitter-wing install",
     "install:wing-api-checker": "npm --prefix apps/wing-api-checker install",
     "install:jsii-docgen": "npm --prefix apps/jsii-docgen install",
-    "install:wing-playground": "npm --prefix apps/wing-playground install",
-    "install:winglang": "npm --prefix apps/wing install",
     "install:sdk": "npm --prefix libs/wingsdk install",
-    "install:hangar": "npm --prefix tools/hangar install"
+    "install:tree-sitter-wing": "npm --prefix libs/tree-sitter-wing install",
+    "install:winglang": "npm --prefix apps/wing install",
+    "install:hangar": "npm --prefix tools/hangar install",
+    "install:wing-playground": "npm --prefix apps/wing-playground install",
+    "install:vscode-wing": "npm --prefix apps/vscode-wing install"
```

Comment:
Alphabetical order perhaps?

Reply:
I attempted to put it in dependency order in case that was part of the thrashing and `run-p` would vaguely respect the order, but after saying that out loud it's silly so yeah I'll alphabetize.